### ISSUE TITLE
[iceprod2] multiple errors in functions.compress/uncompress (Trac #1602)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1594]
Something must have happened, but most of the compression tests are failing, at least on my laptop.

Migrated from https://code.icecube.wisc.edu/ticket/1602
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Something must have happened, but most of the compression tests are failing, at least on my laptop.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] multiple errors in functions.compress/uncompress", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-03-23T20:28:54", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160323T22:08:14: 
* david.schultz changed status from "new" to "accepted"
* david.schultz changed type from "rumor / allegation" to "defect"
* david.schultz commented:

> Summary: remove my excessiveness from the initial months of iceprod2 development.
> 
> Detailed:
> 
>   `7za -mmt=off` no longer works in version 15. There is a similar flag that does the same thing (actually, allows you to enter a number of cores instead of all or none).  But, this means supporting two APIs, which is bad.
> 
>   Rip out `7za` support and replace with venerable `tar`.  It doesn't support as many features, but it's stable and available.


* david.schultz changed owner from "" to "david.schultz"

**Comment by icecube-trac** [comment]
Trac update at 20160328T16:59:44: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "accepted" to "closed"
* david.schultz commented:

> In changeset 143786:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="143786"
> fixing lots of tests, especially related to download / compression. closes #1594
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
